conda_lsst/recipe_maker.py

```diff
@@ -84,7 +84,7 @@ def gen_conda_package(self, product, sha, eups_version, giturl, eups_deps, eups_
 		# the lsst-product-configs package as well, as that's where the .table
 		# and .cfg files are.
 		if depends_on_internal:
-			product_configs = 'lsst-product-configs ==0.0.1'
+			product_configs = 'lsst-product-configs ==1.0.1925'
```

Comment:
Come to think of it, maybe we shouldn't do this as it will be super trivial to update the `lsst-product-config` recipe (the meta.yaml), but forget to update `config.yaml`, and then be confused why the build isn't working.

Instead, it may be better to read the version from `lsst-product-config`'s meta.yaml, e.g., like we do [here](https://github.com/mjuric/conda-lsst/blob/master/conda_lsst/recipe_maker.py#L203).